kuma/settings/common.py

```diff
@@ -836,6 +836,12 @@ def pipeline_one_scss(slug, **kwargs):
         ),
         'output_filename': 'build/styles/signup-flow.css',
     },
+    'auth-modal': {
+        'source_filenames': (
+            'styles/minimalist/components/auth-modal.scss',
```

Comment:
Why is there a generic `modal.js` that is separate from `auth-modal.js` if that's not the case for the SCSS files?

Reply:
It is separate as ` kuma/static/styles/minimalist/components/_modal.scss`. Currently `auth-modal` is the only one using it so it pulls it in directly as a dependency here `kuma/static/styles/minimalist/components/auth-modal.scss`. Once we have two different modals using `_modal.scss` we can/should load it as a separate bundle so we do not include the same CSS twice.